src/main/java/hudson/tasks/test/AbstractTestResultAction.java

```diff
@@ -80,6 +88,27 @@
 
     private Map<String,String> descriptions = new ConcurrentHashMap<String, String>();
 
+    /**
+     * Tool tips for "Overall Build Analysis" trend type.
+     */
+    private Map<NumberOnlyBuildLabel,String> failToolTip, totalToolTip, skipToolTip;
+
+    /**
+     * Tool tips for "Lengthy Tests" trend type.
+     */
+    private Map<NumberOnlyBuildLabel, String> lengthyToolTip;
+
+    /**
+     * Array for storing package hierarchy derived from each testcase, used for selecting a particular
+     * project.
+     */
+    private String[] projectList;
+
+    private List<String> mostFailedTestCases;
```

Comment:
Please initialize

Reply:
Can't initialize them at the time of declaration as they are private variables of an abstract class and won't get initialized on creating an object of implementing class.